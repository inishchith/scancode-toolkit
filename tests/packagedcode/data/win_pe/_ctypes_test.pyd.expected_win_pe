{
  "APIVersion": "", 
  "Assembly Version": "", 
  "BuildDate": "", 
  "BuildVersion": "", 
  "Comments": "", 
  "Company": "", 
  "CompanyName": "", 
  "FileDescription": "", 
  "FileVersion": "", 
  "Full Version": "", 
  "InternalName": "", 
  "LegalCopyright": "", 
  "LegalTrademarks": "", 
  "LegalTrademarks1": "", 
  "LegalTrademarks2": "", 
  "License": "", 
  "OriginalFilename": "", 
  "ProductName": "", 
  "ProductVersion": "", 
  "WWW": "", 
  "extra_data": {}
}