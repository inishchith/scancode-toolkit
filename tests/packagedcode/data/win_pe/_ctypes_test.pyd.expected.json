{
  "APIVersion": null, 
  "Assembly Version": null, 
  "BuildDate": null, 
  "BuildVersion": null, 
  "Comments": null, 
  "Company": null, 
  "CompanyName": null, 
  "FileDescription": null, 
  "FileVersion": null, 
  "Full Version": null, 
  "InternalName": null, 
  "LegalCopyright": null, 
  "LegalTrademarks": null, 
  "LegalTrademarks1": null, 
  "LegalTrademarks2": null, 
  "License": null, 
  "OriginalFilename": null, 
  "ProductName": null, 
  "ProductVersion": null, 
  "WWW": null, 
  "extra_data": {}
}